lib/ecto_nested_changeset.ex

```diff
@@ -213,6 +213,24 @@ defmodule EctoNestedChangeset do
     nested_update(:delete, changeset, path, mode)
   end
 
+  @doc """
+  Returns a value from a changeset referenced by the path.
+
+  The last path segment must be an atom referencing either a to-many relation
+  field or an array field.
```

Comment:
This is only the case for the functions that manipulate lists. For `get_at`, it should be possible to point to any value. Can you remove this part of the docstring and add another test for a path that ends with an index, e.g. `[:pets, 1, :toys, 2]`?